shared-bindings/audioio/AudioOut.c

```diff
@@ -43,13 +43,15 @@
 //|
 //| AudioOut can be used to output an analog audio signal on a given pin.
 //|
-//| .. class:: AudioOut(left_channel, right_channel=None)
+//| .. class:: AudioOut(left_channel, *, right_channel=None, default_value=0x8000)
```

Comment:
I was trying to think of a more descriptive name for this than "default_value". Maybe "silence_value" or "midpoint value"?. @CedarGroveStudios, is there a standard name?

Reply:
The term "quiescent value" is frequently used to describe both the bipolar zero value of the sample stream data and the corresponding unipolar-based value of the DAC. How does that sound to you? @dhalbert

Reply:
`quiescent_value` works for me. I don't expect people to change it very often. Will update it soon.